src/ripple/peerfinder/impl/PeerfinderConfig.cpp

```diff
@@ -26,29 +26,24 @@ namespace PeerFinder {
 Config::Config()
     : maxPeers(Tuning::defaultMaxPeers)
     , outPeers(calcOutPeers())
+    , inPeers(0)
     , wantIncoming(true)
     , autoConnect(true)
     , listeningPort(0)
     , ipLimit(0)
 {
 }
 
-double
+std::size_t
 Config::calcOutPeers() const
 {
-    return std::max(
-        maxPeers * Tuning::outPercent * 0.01, double(Tuning::minOutCount));
+    return std::round(std::max(
+        maxPeers * Tuning::outPercent * 0.01, double(Tuning::minOutCount)));
```

Comment:
Once `maxPeers` is changed to `std::size_t`, this can be rewritten using integer math, saving the overhead of using `double`
```
    return std::max(
        (maxPeers * Tuning::outPercent - 1) / 100 + 1,
        std::size_t(Tuning::minOutCount));
```